tickets/src/main/java/no/unit/nva/publication/ticket/update/UpdateTicketHandler.java

```diff
@@ -201,39 +199,32 @@ private void publishingRequestSideEffects(PublishingRequestCase ticket,
                                               UpdateTicketRequest ticketRequest)
         throws NotFoundException {
         if (CLOSED.equals(ticketRequest.getStatus())) {
-            updateUnpublishedFilesToUnpublishable(ticket.getResourceIdentifier());
+            updatePendingFilesToRejected(ticket.getResourceIdentifier());
         }
         if (COMPLETED.equals(ticketRequest.getStatus())) {
             ticket.approveFiles();
             ticketService.updateTicket(ticket);
         }
     }
 
-    private void updateUnpublishedFilesToUnpublishable(SortableIdentifier publicationIdentifier)
+    private void updatePendingFilesToRejected(SortableIdentifier publicationIdentifier)
         throws NotFoundException {
         var publication = resourceService.getPublicationByIdentifier(publicationIdentifier);
         var updatedPublication = publication.copy()
-                                     .withAssociatedArtifacts(updateUnpublishedFiles(publication))
+                                     .withAssociatedArtifacts(rejectFiles(publication))
                                      .build();
         resourceService.updatePublication(updatedPublication);
     }
 
-    private List<AssociatedArtifact> updateUnpublishedFiles(Publication publication) {
+    private List<AssociatedArtifact> rejectFiles(Publication publication) {
         var associatedArtifacts = publication.getAssociatedArtifacts();
         return associatedArtifacts.stream()
-                   .map(this::rejectIfUnpublished)
+                   .map(file -> file instanceof PendingFile<?> pendingFile
```

Comment:
This appears to only reject pending files. Update the name. Also, extract the inlined code here. `rejectPendingFiles`, `rejectSingleFile`

Reply:
Moved permanently to AcceptedPublishingRequestEventHandler